httpcore/_backends/sync.py

```diff
@@ -187,9 +189,9 @@ def connect_tcp(
         self,
         host: str,
         port: int,
-        timeout: typing.Optional[float] = None,
-        local_address: typing.Optional[str] = None,
-        socket_options: typing.Optional[typing.Iterable[SOCKET_OPTION]] = None,
+        timeout: float | None = None,
+        local_address: str | None = None,
+        socket_options: typing.Iterable[SOCKET_OPTION] | None = None,
```

Comment:
I had the same thought, but some files use `import typing` while others use `from typing import ...`. I kept them in their original format to avoid unnecessary changes. Should I unify the format across all files?

Reply:
> I kept them in their original format to avoid unnecessary changes.

👍

> Should I unify the format across all files?

Possibly, tho let's consider that separately.